src/tests/Microsoft.PowerFx.Core.Tests/TexlTests.cs

```diff
@@ -1483,7 +1483,6 @@ public void TexlFunctionTypeSemanticsVarPTypedGlobalWithCoercion()
         [InlineData("Text(123, \"$000.000\")")]
         [InlineData("Text(123, \"dddd, mm/dd/yy, at hh:mm:ss am/pm\")")]
         [InlineData("Text(\"hello world\")")]
-        [InlineData("Text(\"hello world\", \"mm/dd/yyyy\")")]
```

Comment:
>    [InlineData("Text(\"hello world\", \"mm/dd/yyyy\")")] [](http://example.com/codeflow?start=4&length=57)

Instead of deleting it, we should move it to the negative test.

Reply:
moved this to negative test.